pkgs/by-name/ti/tiramisu/package.nix

```diff
@@ -10,12 +10,11 @@
 stdenv.mkDerivation rec {
```

Comment:
I see other things that can be improved as well:

- `stdenv.mkDerivation rec { }` -> `stdenv.mkDerivation (finalAttrs: { })`
- `meta = with lib; {` -> `meta = {`